app/Services/ModuleService.php

```diff
@@ -50,9 +41,8 @@ public function getFrontendLinks($logged_in): array
      *
      * @param string $title
      * @param string $url
-     * @param string $icon
      */
-    public function addAdminLink(string $title, string $url, string $icon = '')
```

Comment:
Why was the `$icon` param removed?

Reply:
Re-inserted it with default value. Can you check please?

Reply:
Yes looks good now, thanks.

Reply:
Welcome :D